src/user-guide/cheat-sheet.rst

```diff
@@ -221,7 +221,7 @@ Run or re-run a task (:ref:`more info <interventions.re-run-a-task>`)::
 Mark a task as "succeeded"
 (:ref:`more info <interventions.set-task-outputs>`)::
 
-   cylc set <id>//<cycle>/<task>
```

Comment:
Or perhaps instead:
```diff
diff --git a/src/user-guide/cheat-sheet.rst b/src/user-guide/cheat-sheet.rst
index 51c90bf9..7cd7f7f0 100644
--- a/src/user-guide/cheat-sheet.rst
+++ b/src/user-guide/cheat-sheet.rst
@@ -220,6 +220,6 @@ Run or re-run a task (:ref:`more info <interventions.re-run-a-task>`)::
 
-Mark a task as "succeeded"
+Mark a task as complete
 (:ref:`more info <interventions.set-task-outputs>`)::
 
-   cylc set <id>//<cycle>/<task> --out=succeeded
+   cylc set <id>//<cycle>/<task>
```

Reply:
IMO many users don't need to think about completion where completion != succeeded.